src/js/player.js

```diff
@@ -534,7 +535,6 @@ class Player extends Component {
     textTrackConverter.jsonToTextTracks(this.textTracksJson_ || [], this.tech);
 
     this.on(this.tech, 'ready', this.handleTechReady);
-    this.on(this.tech, 'usenativecontrols', this.handleTechUseNativeControls);
```

Comment:
no longer needed because we assume if `techGet('controls')` is true, we are using native controls.